packages/related-keyphrase-suggestions/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@yoast/related-keyphrase-suggestions",
+  "version": "0.1.0",
+  "description": "Yoast related keyphrase suggestions library",
+  "main": "build/index.js",
+  "style": "build/css/style.css",
+  "files": [
+    "build",
+    "!*.map"
+  ],
+  "author": "Team Yoast <support@yoast.com>",
+  "license": "GPL-3.0",
+  "private": false,
+  "sideEffects": false,
+  "scripts": {
+    "clean:build": "rm -rf build",
+    "build": "yarn clean:build && yarn build:js && yarn build:css",
+    "build:js": "babel src --out-dir build --ignore \"src/**/*.stories.js\",\"src/**/stories.js\",\"src/**/docs/**/*.js\"",
+    "build:css": "node ./scripts/build-css.js",
+    "watch": "yarn watch:js & yarn watch:css",
+    "watch:js": "yarn build:js --watch",
+    "watch:css": "node ./scripts/watch-css.js",
+    "storybook": "storybook dev -p 6007",
+    "build:storybook": "storybook build",
+    "lint": "eslint src --max-warnings=0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.23.9",
+    "@babel/core": "^7.23.9",
+    "@babel/eslint-parser": "^7.23.9",
+    "@babel/eslint-plugin": "^7.23.5",
+    "@babel/parser": "^7.23.9",
+    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
+    "@babel/plugin-transform-runtime": "^7.23.9",
+    "@babel/preset-env": "^7.23.9",
+    "@babel/preset-react": "^7.23.3",
+    "@mdx-js/react": "^2.1.5",
+    "@storybook/addon-a11y": "^7.6.17",
+    "@storybook/addon-docs": "^7.6.17",
+    "@storybook/addon-essentials": "^7.6.17",
+    "@storybook/addon-links": "^7.6.17",
+    "@storybook/addon-styling-webpack": "^0.0.6",
+    "@storybook/addons": "^7.6.17",
+    "@storybook/mdx2-csf": "^1.1.0",
+    "@storybook/preview-api": "^7.6.17",
+    "@storybook/react": "^7.6.17",
+    "@storybook/react-webpack5": "^7.6.17",
+    "@storybook/theming": "^7.6.17",
+    "@wordpress/jest-preset-default": "^8.0.1",
+    "@yoast/browserslist-config": "^1.2.3",
+    "@yoast/jest-preset": "^1.0.1",
+    "@yoast/postcss-preset": "^1.2.0",
+    "@yoast/tailwindcss-preset": "^2.3.0",
+    "babel-loader": "^9.1.3",
+    "css-loader": "^6.10.0",
+    "eslint": "^8.57.0",
+    "eslint-config-yoast": "^6.0.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-react": "^7.34.1",
+    "jest": "^27.5.1",
+    "postcss": "^8.4.19",
+    "postcss-loader": "^7.3.4",
+    "puppeteer": "^17.1.3",
+    "storybook": "^7.6.17",
+    "tailwindcss": "^3.2.4",
+    "style-loader": "^3.3.4"
+  },
+  "dependencies": {
+    "classnames": "^2.3.2",
+    "@heroicons/react": "^1.0.6",
```

Comment:
We want to icons from version 1 in the design.

Reply:
> We want to icons from version 1 in the design

Has this been communicated to the UX team? Maybe we can live without those (or upgrade them to a newer version), so that we use the latest library?

Reply:
Yes, I changed it to version 2 and Kai asked to revert